unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
```

Comment:
```
export interface IFormQualifierEntity extends IEntity {
  code: string;
  name: string;
  type: EEntityType.FormQualifier;
  children?: IFormQualifierEntityChildren;
  properties?: IFormQualifierEntityProperties;
}
```

Type is `FormQualifier`, maybe meet in the middle with `DoseFormQualifier`?

Reply:
Works for me!

Reply:
edcf0cef8